docker/kube/Dockerfile

```
@@ -1,35 +1,74 @@
-#{{{ fpm
-FROM europe-west1-docker.pkg.dev/tao-artefacts/base-images/php-v8.4:v1.0.20 as fpm
+# ---------- Stage 1: Composer ----------
+FROM composer:2.8 AS composer
+WORKDIR /app
+COPY . .
+RUN set -eux;
+        composer install -n --no-dev --prefer-dist --optimize-autoloader;
```

Hadolint (2.14.0)

[error] unexpected 'c'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)